app/templates/events/view.hbs

```diff
@@ -1,12 +1,14 @@
-{{#if (or (eq session.currentRouteName 'events.view.index') (eq session.currentRouteName 'events.view.export'))}}
-  {{#if (eq model.state 'draft')}}
-    <div class="ui grid stackable">
-      <div class="row">
-        <div class="sixteen wide column">
-          {{unverified-user-message}}
+{{#if unverified-user-message}}
+  {{#if (or (eq session.currentRouteName 'events.view.index') (eq session.currentRouteName 'events.view.export'))}}
+    {{#if (eq model.state 'draft')}}
```

Comment:
Make the above if merge with this.

Reply:
I am not understanding, what you are saying?can you please explain?

> Make the above if merge with this.

Reply:
I am not understanding, what you are saying?can you please explain?

Reply:
Merge the top `if` with this `if`

Reply:
ok